aptos-move/parallel-executor/src/proptest_types/tests.rs

```diff
@@ -212,3 +211,134 @@ fn module_publishing_fallback() {
         (true, true),
     ));
 }
+
+fn publishing_fixed_params() {
+    let mut runner = TestRunner::default();
+    let num_txns = 300;
+
+    let universe = vec(any::<[u8; 32]>(), 50)
+        .new_tree(&mut runner)
+        .expect("creating a new value should succeed")
+        .current();
+    let transaction_gen = vec(
+        any_with::<TransactionGen<[u8; 32]>>(TransactionGenParams::new_dynamic()),
+        num_txns,
+    )
+    .new_tree(&mut runner)
+    .expect("creating a new value should succeed")
+    .current();
+    let indices = vec(any::<Index>(), 4)
```

Comment:
seems you only use 0 and 2, why this is 4? also why it's not 0 and 1?

Reply:
I use 1 and 3 for indices where I insert the new read and new write, e.g. line 253